docs/docs/migrate/keptnapp/assets/keptnapp-migrated.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: "some-keptn-app"
   namespace: "my-app-ns"
   labels:
-    my-custom-label: customValue
```

Comment:
there's a second occurrence of this under `assets/keptnapp.yaml` that should also be removed

Reply:
done